src/main/resources/attackvectors/BlindSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,29 @@
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_1=As blind SQL injection doesn't show any internal details like errors or table data hence it allows a hacker to check if the web application is vulnerable to SQL injection by sending <b>true</b> and <b>false</b> statements.<br/>\
+ <b>How to exploit this level?</b><br/>\
+  Insert those in textbox<br/>\
+  <u>For true statement</u><br/>\
+  <code>100 OR 2=2</code><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": true }</i><br/>\
+We know <i>100 id</i> is not present in the table but still, it is returning <i>true</i>. it means that the input is evaluated to <i>true</i> and the backend has returned that <i>car</i> is present, which could indicate vulnerability.<br/>\
+  <u>For false statement</u><br/>\
+  <code>1 AND 1=2</code><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": false }</i> <br/>\
```

Comment:
I think <code> tag looks better in UI. What do you think ?

Reply:
You are right, I will change italic to code tag